agent/consul/server.go

```diff
@@ -341,6 +342,9 @@ type Server struct {
 	// Manager to handle starting/stopping go routines when establishing/revoking raft leadership
 	leaderRoutineManager *routine.Manager
 
+	// publisher is the precreated publisher
+	publisher *stream.EventPublisher
```

Comment:
nit: This comment feels like it could use a little more context. What do you think of mentioning the kinds of events that are published, and which components publish or subscribe to it?

Reply:
Done